orchestration/dashboard.py

```
@@ -0,0 +1,361 @@
+        print("      ./orchestration/monitor_agents.sh")
+
+        # System Resources
+        print("\n💾 SYSTEM RESOURCES:")
+
+        # Check Redis
+        try:
+            redis_result = subprocess.run(
+                ["redis-cli", "ping"], check=False, capture_output=True, text=True
+            )
+            redis_status = "🟢 ONLINE" if redis_result.returncode == 0 else "🔴 OFFLINE"
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)